src/main/java/com/ibm/crypto/plus/provider/EdDSAPrivateKeyImpl.java

```diff
@@ -90,13 +90,13 @@ public EdDSAPrivateKeyImpl(OpenJCEPlusProvider provider,
             }
 
             if (this.key == null) {
-                int pub_size = CurveUtil.getPublicCurveSize(curve);
+                int pub_size = CurveUtil.getCurveSize(curve);
                 this.xecKey = XECKey.generateKeyPair(provider.getOCKContext(),
                         this.curve.ordinal(), pub_size);
             } else {
                 this.algid = CurveUtil.getAlgId(this.curve);
                 byte[] der = buildOCKPrivateKeyBytes();
-                int priv_size = CurveUtil.getPrivateCurveSize(curve);
+                int priv_size = CurveUtil.getDEREncodingSize(curve);
```

Comment:
Should we change the name of the variable to `encodingSize`?

Reply:
Given that this really does represent an encoding I am ok using `encodingSize` as the variable name here.